src/main/target/FLIGHTONE/config.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#include "drivers/serial.h"
+#include "rx/rx.h"
+
+#include "telemetry/telemetry.h"
+
+#include "fc/config.h"
+
+
+#ifdef USE_TARGET_CONFIG
+
+#include "pg/max7456.h"
+#include "pg/pg.h"
+
+
+void targetConfiguration(void)
+{
+    rxConfigMutable()->halfDuplex = true;
+    serialConfigMutable()->portConfigs[1].functionMask = FUNCTION_RX_SERIAL;
```

Comment:
There's a better way to set it up in `target.h`:
```
#define SERIALRX_PROVIDER       SERIALRX_SBUS
#define SERIALRX_UART           SERIAL_PORT_USART1
```

Reply:
Is there a way to enable half duplex through defines or is using the target config the only way to do that?

Reply:
Runtime target configuration is the only way for now, it's ok to use it

Reply:
Ok sounds good ill leave that line in and remove the other then